beacon/goclient/signing.go

```diff
@@ -1,7 +1,10 @@
 package goclient
 
 import (
+	"context"
 	"crypto/sha256"
+	"fmt"
+	"github.com/attestantio/go-eth2-client/api"
```

Comment:
nit: I think this should be near "github.com/attestantio/go-eth2-client/spec/phase0" to be able to quickly see the scope of all imports from the library